### ISSUE TITLE
The UIAccessibility informal protocol should be implemented by all "standard" UIKit views and controls

### ISSUE THREAD

**Issue by BlueRiverInteractive** [robovm/robovm#948]
There's no documentation about what UIKit views and controls implement that protocol. 
We can find out if a class conforms to UIAccessibility by using `conformsToProtocol()`.

**Comment by BlueRiverInteractive** [comment]
EVERY NSObject responds to the UIAccessibility methods. Putting all those accessibility methods into NSObject sounds a bit too much. I would propose to only let UIResponder implement that protocol. This way all ui views and controls will be accessible and NSObject won't be trashed.
